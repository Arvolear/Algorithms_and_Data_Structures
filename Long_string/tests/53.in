171625467537641694901857060
16945320068811775835257552250481205857853656