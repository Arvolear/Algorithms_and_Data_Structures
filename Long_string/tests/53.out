16945320068811776006883019788122900759710716
16945320068811775663632084712839510955996596
2908248479384803765275808735499459096483917576651878716151940110411360
98734298073188057